### PR TITLE
Submitted UI library blog post

### PR DESCRIPTION
**Pull request by gneatgeek**

### CODE REVIEW

**Comment by anelson425** [comment]
Looks good @gneatgeek 

@kpshek can you merge this in please?

File: source/_posts/2016-10-13-building-a-unified-ui-component-library.markdown [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Blue Steel is a Rails gem that provides some basic helpers, CSS, JavaScript, and site templates to simplify layout and correct HTML usage. Blue Steel abstracts complex components to keep things simple and because we didn’t feel it was necessary to abstract native HTML elements.
+
+In its infancy, the project was well received; other platforms saw the value and wanted to take advantage of the work being done. To accommodate the additional platforms, Blue Steel was merged with another internal UI library called Style Guide. The best features of each library were pulled together, sometimes one overriding the other, sometimes merging feature sets. The merged project was called Terra.
+
+## Problems
+The approach of merging existing components saved time up-front, but came with a steep cost. The library was fragmented in the approaches taken to design and development. As such, it became difficult to work on without pre-existing familiarity. It was fairly obvious that we put two frameworks together on a time constraint.
```
Comment (mikeantonelli) [inline source/_posts/2016-10-13-building-a-unified-ui-component-library.markdown]: `taken to design and develop` or `taken during design and development`.

**Comment by mikeantonelli** [comment]
+1

File: source/_posts/2016-10-13-building-a-unified-ui-component-library.markdown (hunk 1 above)
Comment (gneatgeek) [inline source/_posts/2016-10-13-building-a-unified-ui-component-library.markdown]: Good catch!

**Comment by kpshek** [comment]
Merged and deployed: http://engineering.cerner.com/blog/building-a-unified-ui-component-library/

Note that I made an edit of my own during the merge in f10d056: replacing the Cerner-ism "non-passive" with the term everyone outside of Cerner will understand: "breaking change". I also removed the permalink; that's only necessary if the blog URL differs from the name markdown file.

**Pull request merged**